scripts/start.sh

```diff
@@ -5,9 +5,24 @@ set -e
 CUR_DIR=$(cd "$(dirname "$0")" && pwd)
 BASE_DIR=$(dirname "$CUR_DIR")
 
+min() {
```

Comment:
From a global point of view, I think we should keep the possibility to use gunicorn.
It stays a legitimate tool, even if we change our internal deployment to not use it anymore.

This will also allow us to transition more easily between the 2 kind of deployments: we will be able to configure in deployment what kind of load balancing we use.

In order to achieve this, we could add an argument to the script, or an env variable, like `--no-gunicorn` or `--multiple-ports` which would trigger the new behaviour.

Also we need to check if the README and other parts of the developer documentation needs to be updated, as well as the provided `docker-compose.yaml` in the root directory.